spec/features/admin/dashboard_spec.rb

```diff
@@ -48,8 +48,8 @@
     it "displays links to merchant and admin invoices indices" do
       visit admin_index_path
 
-      expect(page).to have_link("Merchants")
-      expect(page).to have_link("Invoices")
+      expect(page).to have_link("Merchants") # Is this robust enough or should 
+      expect(page).to have_link("Invoices")    # these be liketests for US#22?
```

Comment:
I think this is great for testing the presence of these links on the admin index page. We would add to this if we wanted to test that that these links bring us to other pages utilizing capybara's `click_link` syntax.